flexmeasures/ui/utils/color_defaults.py

```diff
@@ -10,7 +10,7 @@ def get_color_settings(account: Account | None) -> dict:
     """
 
     primary_color: str = "#1a3443"
-    secondary_color: str = "f1a122"
+    secondary_color: str = "#f1a122"
```

Comment:
Looks like a bug potentially worth a separate PR in case this PR still takes a while to get over the fence.

Reply:
This PR will not take long.
I confirmed that it works, we have only a few things to go. No need for a spearate PR I think.